.github/workflows/release.yml

```diff
@@ -130,17 +130,17 @@ jobs:
           context: .
           platforms: linux/amd64
           push: true
-          tags: quay.io/sustainable_computing_io/kepler:latest, quay.io/sustainable_computing_io/kepler:${{ github.event.inputs.release }}
+          tags: quay.io/sustainable_computing_io/kepler:latest, quay.io/sustainable_computing_io/kepler:${{ github.event.inputs.release }}-bcc
```

Comment:
should we have the `latest` tag here?

Reply:
lets take this question in some other PR

Reply:
The reason I ask is because I am assuming that the previous release tagged `bcc` as latest and now we want to switch that to libbpf. Would that be the case if we add `latest` here?

Reply:
`tags` field for both images has `quay.io/sustainable_computing_io/kepler:latest`